tests/core/ckeditor/manual/hcdetection.html

```diff
@@ -20,12 +20,17 @@
 </div>
 
 <script>( function() {
-	CKEDITOR.replace( 'editor' );
 
 	var indicator = CKEDITOR.document.getById( 'indicator' ),
 		state = CKEDITOR.env.hc ? 'on' : 'off'
 
 	indicator.addClass( state );
 	indicator.setText( 'HC is ' + state );
+
+	if(CKEDITOR.env.safari || CKEDITOR.env.chrome || CKEDITOR.env.mobile) {
```

Comment:
Please do the following:
1. Add missing spaces before and after brackets.
2. Move this `if` statement to the beginning of the `<script>` block - we want to know the test should be ignored as soon as possible.

Reply:
:thinking: I should explain myself. I decided to move ignore:

*   after HC indicator: just in case it will be green in the future, and the test still will be ignored: we will be able to notice this incorrectness.
*   before editor creation: to not create editor in ignore test.

Reply:
Thanks for explanation - it would be viable idea, but when you run series of tests from dashboard, the ignored ones are skipped completely (they don't blink for a second, they just don't show up at all) and tester doesn't have a chance to see that the button is green. That's why the case with a dev changing the test in the future and seeing the ignore as the first thing is more probable 🙂

Reply:
As for the second part:

> before editor creation: to not create editor in ignore test.

This wouldn't normally change anything (you can check in other tests that the code after `bender.ignore()` is still executed), but since you enclosed the rest of JS in IIFE, it indeed stops right there and moves to the next test faster (we gained about 100ms in each test on my machine), which is cool 👍🏻  I've checked all the browsers and everything works fine.